### PR TITLE
implemented backend apiclient and startgame on game controller

### PR DESCRIPTION
**Pull request by gmckz**
Added apiClient and GameController with startGame function

### CODE REVIEW

File: api/controllers/game.js
Comment (LydiaNtafa) [inline api/controllers/game.js:1]: please push chatGpt response to chatGPTMessages

File: api/routes/index.js
Comment (LydiaNtafa) [inline api/routes/index.js:1]: change the GET to POST

**Review by gmckz — COMMENTED** [review]
are the changes here?

**Review by LydiaNtafa — APPROVED** [review]
look great to me!

**Pull request merged**